test/functional/mempool_accept_v3.py

```diff
@@ -201,21 +217,47 @@ def test_nondefault_package_limits(self):
         """
         node = self.nodes[0]
         self.log.info("Test that a decreased limitdescendantsize also applies to v3 child")
-        tx_v3_parent_large1 = self.wallet.send_self_transfer(from_node=node, target_weight=99900, version=3)
-        tx_v3_child_large1 = self.wallet.create_self_transfer(utxo_to_spend=tx_v3_parent_large1["new_utxo"], version=3)
-        # Child is within v3 limits, but parent's descendant limit is exceeded
-        assert_greater_than(1000, tx_v3_child_large1["tx"].get_vsize())
+        parent_target_weight = 9990 * WITNESS_SCALE_FACTOR
+        child_target_weight = 500 * WITNESS_SCALE_FACTOR
+        tx_v3_parent_large1 = self.wallet.send_self_transfer(
+            from_node=node,
+            target_weight=parent_target_weight,
+            version=3
+        )
+        tx_v3_child_large1 = self.wallet.create_self_transfer(
+            utxo_to_spend=tx_v3_parent_large1["new_utxo"],
+            target_weight=child_target_weight,
+            version=3
+        )
+
+        # Parent and child are within v3 limits, but parent's descendant limit is exceeded
```

Comment:
<s>I don’t understand the test here. My understanding was that TRUC transactions would be generally limited to 10 kvB, and TRUC transactions with a parent would be limited to 1 kvB. It seems to me that they should together be allowed to weigh up to 11 kvB, so it is no clear to me why this comment reads "but parent's descendant limit is exceeded".</s>

Perhaps clarify here in the comment that there is a custom descendant limit in play:
```suggestion
        # Parent and child are within v3 limits, but the reduced descendant limit is exceeded for the parent
```

Reply:
done